File: camunda-service/src/main/java/ru/taratonov/camundaservice/model/LoanRequest.java
package ru.taratonov.camundaservice.model;

import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class LoanRequest {
    private Long id;
    private String name;
    private String surname;
    private Integer age;
    private Integer dependentAmount;
    private Integer salary;
    private Boolean result;
}

File: main-service/src/main/kotlin/ru/taratonov/mainservice/model/LoanRequest.kt
package ru.taratonov.mainservice.model

data class LoanRequest(
    val id: Long? = null,
    val name: String? = null,
    val surname: String? = null,
    val age: Int? = null,
    val dependentAmount: Int? = null,
    val salary: Int? = null,
    val result: Boolean? = null
)
